cupyx/fallback_mode/data_transfer.py

```diff
@@ -2,11 +2,12 @@
 Data transfer methods
 """
 
-import cupy as cp
 import numpy as np
 
+import cupy as cp
+
 
-def _get_xp_args(org_module, to_xp, arg):
+def _get_xp_args(ndarray_instance, to_xp, arg):
```

Comment:
`ndarray_instance` is an instance of the original ndarray whereas `to_xp` is a function which converts original ndarray to the target. This mismatch of type seems not intuitive for us.  How about adding a document about the type of arguments?

Reply:
Yeah, will do that.